### PR TITLE
Draft pick function update

### PR DESCRIPTION
**Pull request by PolinaRomanchenko**
Added functionality for draft_pick function

*--TODO--*
Still need to add more error handling tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +19,7 @@ both relevant and historical statistics from the publicly available but
 as of yet undocumented NHL.com API. As of now the package has limited
 functionality, and is considered a work-in-progress. We will add in
 additional functionality in the coming weeks which can easily be
```
Comment (chuusan) [inline README.md]:
```suggestion
functionality, and is considered as a work-in-progress package. We will add additional functionalities in the coming weeks which can easily be
```

File: README.md [hunk 2]
```diff
@@ -19,8 +19,7 @@ both relevant and historical statistics from the publicly available but
...
-extended based on feedback we
-recieve.
+extended based on feedback we recieve.
```
Comment (chuusan) [inline README.md]:
```suggestion
extended based on the feedbacks we receive.
```

**Review by chuusan — CHANGES_REQUESTED** [review]
codes looks fine, but typo should be modified.

**Review by chuusan — APPROVED** [review]

**Review by PolinaRomanchenko — COMMENTED** [review]
Thanks for the review!

**Pull request merged**